web/src/main/java/org/openmrs/web/filter/update/UpdateFilter.java

```diff
@@ -9,17 +9,38 @@
  */
 package org.openmrs.web.filter.update;
 
-import liquibase.changelog.ChangeSet;
-import liquibase.exception.LockException;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.LinkedList;
```

Comment:
Can you discard the changes to these import statements?

Reply:
@dkayiwa  I tried to re-run mvn, to update the project and force update snapshots, and also to save actions with the java editor but failed. Could you advise accordingly?

Reply:
@gracebish these are format changes which are brought about when you save your changes ,,
to remove them press `Ctrl+z `immediately after pressing the save button to save your changes in your IDE ,

Reply:
@gitcliff the issue is not with trailing spaces, it is with imports which come about after making changes. I wonder how to deal with these imports.

Reply:
yes @gracebish its what i was trying to explain earlier press `Ctrl+z `immediately after saving your changes and these un-necessary imports will be reverted

Reply:
@gracebish did you see the above from cliff?